node_modules/node-red-node-email/61-email.html

```diff
@@ -84,7 +88,8 @@
             dname: {value:""},
             to: {required: true},
             from: {required: true},
-            subject: {required: true}
+            subject: {required: true},
+            body : {required: true}
```

Comment:
To keep record of off-review talk: this field is just a pointer to the variable holding the body.